examples/evm-to-evm-fungible-transfer/package.json

```diff
@@ -32,6 +32,6 @@
     "@buildwithsygma/evm": "workspace:^",
     "ethers": "5.7.2",
     "tsx": "^4.15.4",
-    "web3-providers-http": "^1.2.7"
+    "web3-providers-http": "1.10.4"
```

Comment:
Is there any particular reason for using version 1? The current major version is 4 >